examples/README.md

```diff
@@ -0,0 +1 @@
+# AMDMIGraphX Usage Examples
```

Comment:
Can we can add some introduction words here, then gave a link to the readme in each of the subfolder?

Reply:
Maybe we can also add an example using our C/C++ API interface. I attached an example using the C/C++ here. 
[mgx_c_api.tar.gz](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/files/5446293/mgx_c_api.tar.gz)